learning-objectives/intl/pt.yml

```
@@ -1291,3 +1291,45 @@ typescript/decorators:
   links:
     - title: Documentação oficial do Typescript
       url: https://www.typescriptlang.org/docs/handbook/decorators.html
+
+ai-prompting:
+  title: IA Prompting
```

```suggestion
  title: AI Prompting
```

Lo mismo, lo dejaría en inglés para evitar el ingléspañol